src/api-routes/persons.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-unused-vars */
```

Comment:
This file is leftover from the original purpose of this ticket. Felt like a waste to remove it since I'll be implementing it next.